website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -145,43 +144,47 @@ Advanced users can configure multiple projects to use the same Cloud CLI executa
 
 <TabItem value="pip" label="Existing dbt Core users (pip)">
 
+If you already have dbt Core installed, you can install the dbt Cloud CLI using pip. Here are some considerations:
+
+- **Prevent conflicts**<br/>
+  To prevent overwriting dbt Core, avoid installing the dbt Cloud CLI with `pip`. Instead, consider using the native installation method (Homebrew) and configure your PATH or create a new virtual environment.<br/>
+- **Use both dbt Cloud CLI and dbt Core**<br/>
+  Have both the dbt Cloud CLI and dbt Core installed simultaneously to meet your needs. To avoid conflicts, alias the dbt Cloud CLI as "dbt-cloud." For more details, check the [FAQs](#faqs) if your operating system experiences path conflicts.<br/>
```

Comment:
I think this should be "if you use brew, consider aliasing the cloud cli to avoid conflicts". If you install w pip, it will already have overwritten dbt, so this doesn't work there. I would move this to the brew section.